scripts/updateUsers.ts

```
@@ -59,158 +71,195 @@ async function main() {
+          hashrate1hr: convertHashrate(userData.hashrate1hr.toString()).toString(),
+          hashrate1d: convertHashrate(userData.hashrate1d.toString()).toString(),
+          hashrate7d: convertHashrate(userData.hashrate7d.toString()).toString(),
+          lastShare: BigInt(userData.lastshare).toString(),
+          workerCount: userData.workers,
+          shares: BigInt(userData.shares).toString(),
+          bestShare: parseFloat(userData.bestshare),
+          bestEver: BigInt(userData.bestever).toString(),
+        });
+
+        // Extract worker data for each user
+        for (const workerData of (userData.worker || [])) {
```

ESLint

[error] Replace `(userData.worker·||·[])` with `userData.worker·||·[]`

(prettier/prettier)